Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Calendar/Classes/CalendarRenderer.cs

```diff
@@ -1443,37 +1443,40 @@ public virtual void OnDrawTimeScale(CalendarRendererEventArgs e)
             {
                 CalendarTimeScaleUnit? unit = e.Calendar.Days[0]?.TimeUnits[i];
 
-                if (!unit.Visible)
+                if (unit != null && !unit.Visible)
                 {
                     continue;
                 }
 
-                string hours = unit.Hours.ToString("00");
-                string minutes = unit.Minutes == 0 ? "00" : string.Empty;
-
-                if (!string.IsNullOrWhiteSpace(minutes))
+                if (unit != null)
```

Comment:
not needed as it is tested directly above.

Reply:
Rolled back